plugins/woocommerce/src/Blocks/BlockTypes/ProductFilterAttribute.php

```diff
@@ -121,20 +121,18 @@ function ( $item ) use ( $product_attributes_map ) {
 			if ( empty( $params[ "filter_{$product_attribute}" ] ) ) {
 				continue;
 			}
-			$terms           = explode( ',', $params[ "filter_{$product_attribute}" ] );
-			$attribute_label = wc_attribute_label( "pa_{$product_attribute}" );
+			$terms                = explode( ',', $params[ "filter_{$product_attribute}" ] );
+			$attribute_label      = wc_attribute_label( "pa_{$product_attribute}" );
+			$attribute_query_type = $params[ "query_type_{$product_attribute}" ] ?? 'or';
 
 			// Get attribute term by slug.
 			foreach ( $terms as $term ) {
 				$term_object = get_term_by( 'slug', $term, "pa_{$product_attribute}" );
 				$items[]     = array(
-					'type'      => 'attribute',
-					'value'     => $term,
-					'label'     => $attribute_label . ': ' . $term_object->name,
-					'attribute' => array(
-						'slug'      => $product_attribute,
-						'queryType' => 'or',
-					),
+					'type'               => 'attribute/' . $product_attribute,
+					'value'              => $term,
+					'activeLabel'        => $attribute_label . ': ' . $term_object->name,
```

Comment:
Why you hatin' on interpolated strings? 😂

Reply:
Haha, I was quickly testing something and forgot edit. Addressed in 2de833762bf4.